src/utils/LogoMap.ts

```diff
@@ -0,0 +1,14 @@
+import AnywayImage from '../assets/anyway.png';
+import OrYarokLogo from '../assets/Or_Yarok.png';
+
+
+const orYarok = 'אור ירוק'
+const none = null;
```

Comment:
why setting a const to null? simply use null (which in this case, also not required)